packages/app-builder/src/services/editor/ast-editor.tsx

```diff
@@ -315,16 +317,56 @@ export function useRootOrAndChildValidation(stringPath: string) {
   ]);
 }
 
-export function useValidationStatus(stringPath: string) {
+export function useValidationStatus(
+  stringPath: string,
+  returnValue?: ReturnValue,
+) {
   const path = usePath(stringPath);
   const evaluationErrors = useEvaluationErrors(stringPath);
   const parentPath = useParentPath(path);
+  const parentNode = useParentAstNode(parentPath);
+  const isDivByZeroField =
+    isDivisionDenominator(parentNode, path) &&
+    !returnValue?.isOmitted &&
+    returnValue?.value === 0;
```

Comment:
It will be computed for each render cycle, on each node of the tree (since there is a high probability this is used in a recursive comp structure).

Still, since this return a primitive value, it won't make the above `useMemo` recompute.

Reply:
I see, though I was under the impression that this should work fine - no heavy computation involved.
Would you memoize it ?